src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -52,6 +52,12 @@
 (def SemVerVersion
   (schema/pred semver/valid-format? "semver"))
 
+(def StatusProxyConfig
+  {:proxy-target-url schema/Str
```

Comment:
I don't think there is. I'm fine with leaving this how it is now, then, in the interest of giving the user a useful error message.